src/nikhilMJun20/PrimeNumbers.java

```diff
@@ -0,0 +1,51 @@
+package nikhilMJun20;
+
+import java.util.Scanner;
+
+public class PrimeNumbers {
+	void isPrime(int num) {
+		boolean flag = true;
+		for(int div=2; div<=Math.sqrt(num); div++) {
+			if(num % div == 0) {
+				flag = false;
+				break;
+			}
+		}
+		if(flag == true)
+			System.out.print(num+" is a Prime Number.");
+		else
+			System.out.println(num+" is NOT a Prime Number.");
+	}
+	
+	void findPimNumbers() {
```

Comment:
Here you just created one method 
void findPrimeNumberInGivenRange(int startNumber, int endNumber) {
		for(int index=startNumber; index<=endNumber; index++) {
			isPrime(index);
		} 
And performed all operations like sum, count in isPrime() method, and create two instance variables for the same.

Reply:
Hi @KrishnaKTechnocredits,
Made the necessary changes to the code.
Thanks!